### PR TITLE
2183 update guides

### PR DESCRIPTION
**Pull request by mjstahl**
List of changes: https://docs.google.com/a/jupiterjs.com/spreadsheets/d/1pErqpcXVEaPmRFtpqidVBhx9ceP6Ky3HQlWYhbVz1ic/edit?usp=sharing

- [x] Go through each page and check text/code examples
- [x] Check/Update each JSBin
- [x] Update code archives with changes

### CODE REVIEW

**Comment by mjstahl** [comment]
One JSBin needs to be changed to ensure consistency with routing in a template:

http://jsbin.com/surokag/edit?html,js,output

```
<a href="{{routeUrl page='restaurants'}}">
```
should be changed to:

```
<a href="{{ routeUrl(page='restaurants') }}">
```
And

```
<a href="{{routeUrl page='home'}}">
```
should be changed to:

```
<a href="{{ routeUrl(page='home') }}">
```

File: guides/examples/PlaceMyOrder/chapter_3/components/order/order.stache [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 <li>
                   <label>
                     <input type="checkbox"
-                      (change)="{order.items.toggle this}"
+                      ($change)="{order.items.toggle this}"
```
Comment (mjstahl) [inline guides/examples/PlaceMyOrder/chapter_3/components/order/order.stache]: should this be changed to 

```
($change)="order.items.toggle(this)"
```

File: guides/examples/PlaceMyOrder/chapter_7/components/order_phone/order_phone.stache [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="form-group{{#if error}} has-error{{/if}}">
   <label>Phone:</label>
-  <input name="phone" type="text" (keyup)="{setPhoneValue @element.val}">
+  <input name="phone" type="text" {($keyup)}="setPhoneValue(@element.val)">
```
Comment (mjstahl) [inline guides/examples/PlaceMyOrder/chapter_7/components/order_phone/order_phone.stache]: Should this be

```
<input name="phone" type="text" {($keyup)}="setPhoneValue(%element.val)">
```

**Comment by daffl** [comment]
This seems like a lot of additions. Should that still live in our main repository?

**Comment by justinbmeyer** [comment]
Yes. It was worse when it didn't.

**Pull request merged**